src/lib/core/styles/tokens/popover/_tokens.scss

```diff
@@ -28,22 +35,34 @@ $tokens: (
 
   // Animation
   animation-timing: utils.module-val(popover, animation-timing, animation.variable(easing-decelerate)),
```

Comment:
Should we name the animation tokens so it's clear which transitions they apply to? For example, does `zoom-duration` affect entrance *and* exit durations or just entrance durations? Does `zoom-exit-timing` inherit from it? Why isn't there a `zoom-entrance-duration`? It's easy to find out by looking at this file but typical users might be confused

Reply:
Yeah good call. I'll rename those so they specify "enter" in the name of the token now that we have both enter and exit tokens.